modules/events_archive/EED_Events_Archive.module.php

```diff
@@ -287,11 +273,10 @@ public static function link_target()
 
 
     /**
-     *    template_include
-     *
-     * @access    public
```

Comment:
this code is old! 
we used to have to write the function name and the scope (public, private, etc) in the phpdocs because earlier IDEs and code editors could not parse that info from the code itself. Totally not necessary anymore